CONTRIBUTING.md

```diff
@@ -148,19 +150,19 @@ Follow these steps to start contributing ([supported Python versions](https://gi
    $ git checkout -b a-descriptive-name-for-my-changes
    ```
 
-   **Do not** work on the `main` branch.
+   🚨 **Do not** work on the `main` branch!
 
 4. Set up a development environment by running the following command in a virtual environment:
 
    ```bash
    $ pip install -e ".[dev]"
    ```
 
-   (If transformers was already installed in the virtual environment, remove
+   If 🤗 Transformers was already installed in the virtual environment, remove
    it with `pip uninstall transformers` before reinstalling it in editable
-   mode with the `-e` flag.)
+   mode with the `-e` flag.
 
-   To run the full test suite, you might need the additional dependency on `datasets` which requires a separate source
+   To run the full test suite, you might need the `datasets` dependency which requires a separate source
```

Comment:
This is no longer true and will be installed via the `pip install -e ".[dev]", so the whole paragraph can be removed.